backend/Perflow/Common/DTO/Users/UserReadDTO.cs

```diff
@@ -11,6 +11,11 @@ public class UserReadDTO
         public string Email { get; set; }
         public DateTimeOffset Birthday { get; set; }
         public bool Gender { get; set; }
+        public string Description { get; set; }
+        public string Country { get; set; }
+        public string Password { get; set; }
+        public string Salt { get; set; }
```

Comment:
Why do you use Salt and Password in dto?

Reply:
It isn't my class, I've just changed existing records to class.